### ISSUE TITLE
MetArt Network Module Failing with 'variable 'url' referenced before assignment' Error

### ISSUE THREAD

**Issue by albyofdoom** [PAhelper/PhoenixAdult.bundle#877]
Site: https://www.alsscan.com/model/amai-liu/movie/20100605/RED_DEVIL

Log: [com.plexapp.agents.phoenixadult.log](https://github.com/PAhelper/PhoenixAdult.bundle/files/4943319/com.plexapp.agents.phoenixadult.log)

Search Term(s):
ALS Scan - 2010-06-05 - Red Devil
ALS Scan - 2010-06-05 - RED_DEVIL
ALS Scan - 2010-06-05 - Amai Liu - Red Devil

In networkworkMetArt.py, the first few lines look to be the issue. Per the error message, it's attempting to call the 'url' variable before it's been populated. There's also some circular logic, in the that the 'url' variable is eventually populated by items generated from the 'searchResults' variable, which is itself popluated from a JSON call on the 'url' variable.

```
def search(results, encodedTitle, searchTitle, siteNum, lang, searchDate):
    req = PAutils.HTTPRequest(url)
    searchResults = req.json()
```
```
        url = searchResult['item']['path'].rsplit('/', 2)
```

**Comment by DirtyRacer1337** [comment]
Fixed in 16a9996b21f855adffe8c8c29c05533dd06a3277

**Comment by albyofdoom** [comment]
> 
> 
> Fixed in [16a9996](https://github.com/PAhelper/PhoenixAdult.bundle/commit/16a9996b21f855adffe8c8c29c05533dd06a3277)

Confirmed, no further issues. Thanks for the quick fix! :)
